packages/camera/camera/ios/Classes/FLTThreadSafeFlutterResult.h

```diff
@@ -5,7 +5,8 @@
 #import <Flutter/Flutter.h>
 
 /**
- * Wrapper for FlutterResult  that always delivers the result on the main thread.
+ * A thread safe wrapper for FlutterResult that can be called from any thread, by dispatching its
+ * underlying engine APIs to the main thread.
```

Comment:
Nit for all of these class comments: "by dispatching underlying engine calls to the main thread". These classes don't have engine APIs, they only have calls to the engine's APIs.